ElectronicObserver/Utility/Data/EquipmentUpgradeCost.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using ElectronicObserver.Window.Tools.EquipmentUpgradePlanner;
+using ElectronicObserverTypes;
+using ElectronicObserverTypes.Serialization.EquipmentUpgrade;
+
+namespace ElectronicObserver.Utility.Data;
+public static class EquipmentUpgradeCost
```

Comment:
`EquipmentUpgradeCostExtensions` might be a better name here, since the class only exists for providing the extension methods.

Reply:
No idea why i didn't do that from the start